HelpMyStreetFE/HelpMyStreetFE/src/js/account/login.js

```diff
@@ -34,8 +34,7 @@ export const login = async (email, password) => {
             if (authResp.fetchResponse != fetchResponses.SUCCESS) {
                 throw ({ code: "server" });
             }
-
-            if (returnUrl && returnUrl.startsWith("/")) {
```

Comment:
I think we need to keep the && ...

Reply:
(Without that, an attacker could direct someone to a link, which is our login URL plus a return URL starting https:// which they are hosting.  A victim could then click the link, sign in with us, and then believe they are interacting with out site - when in fact they are somewhere else entirely.)